macos/xcode/FluentUI.xcodeproj/project.pbxproj

```diff
@@ -689,13 +728,21 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 8F69C7B22295FABD009F69C0 /* FluentUI_testapp.xcconfig */;
 			buildSettings = {
+				LD_RUNPATH_SEARCH_PATHS = (
```

Comment:
Can we remove these from the xcodeproject and move them into the xcconfig for the test app explicitly?

Reply:
Agreed- Saad it should be a lot easier than what we did in FURN since this is just a regular xcodeproj and not a generated workspace.

Reply:
we should eventually extract our script that fails if any buildSettings are defined in the project.pbxproj so we can use it in open source. Adding that to a long list of things to do

Reply:
Looks like these search paths are already in our xcconfig =)
@anandrajeswaran feel free to add that issue somewhere